### ISSUE TITLE
Implement fromOption support

### ISSUE THREAD

**Issue by ratoshniuk** [7mind/izumi#488]
it would be nice to have methods for Option

```scala
def fromOption[T](opt: Option[T]) : F[Nothing, T]
def fromOptionWith[E](opt: Option[T])(exception: E) : F[E, T]
```

**Comment by pshirshov** [comment]
@kaishh

**Comment by neko-kai** [comment]
@ratoshniuk fromOption is included in current version (0.6.33) https://github.com/pshirshov/izumi-r2/blob/develop/fundamentals/fundamentals-bio/src/main/scala/com/github/pshirshov/izumi/functional/bio/BIOInvariant.scala#L57
